common/lib/xmodule/xmodule/js/src/video/035_video_accessible_menu.js

```diff
@@ -263,18 +278,18 @@ function () {
     function _bindHandlers(state) {
         var menu = state.videoAccessibleMenu;
 
-        // Attach various events handlers to menu container.
         menu.container.on({
-            'mouseenter': _openMenuHandler.bind(menu),
-            'mouseleave': _closeMenuHandler.bind(menu),
             'click': _toggleMenuHandler.bind(menu),
             'keydown': _keyDownHandler.bind(menu)
         });
 
-        // Attach click and keydown event handlers to individual menu items.
         menu.menuItems
-            .on('click', 'a.a11y-menu-item-link', _clickHandler.bind(menu))
-            .on('keydown', 'a.a11y-menu-item-link', _keyDownHandler.bind(menu));
+            .on('click', '.action', _clickHandler.bind(menu))
+            .on('keydown', '.action', _keyDownHandler.bind(menu));
+
+        $(document).on({
```

Comment:
I'm not sure. This is listening to any clicks on the page, and if any of them aren't on the dropdown menu, it'll close the menu. This is actually really important because closing the menu was one of the biggest complaints with the current setup.